java/portal-user-info/README.md

```diff
@@ -1,49 +1,41 @@
-# Portal User Info
-Use the `DefaultAuthenticationChallengeHandler` class to take care of showing an authentication dialog for logging into a `Portal`. Once authenticated against the portal, the app displays information about the authenticated user's profile.
+# Portal user info
 
-![Portal User Info App](portal-user-info.png) 
+Retrieve a user's details via a Portal.
+
+![Image of access portal user info](portal-user-info.png)
+
+## Use case
+
+This portal information can be used to provide a customized UI experience for the user. For example, you can show a thumbnail next to their username in the header of an application to indicate that they are currently logged in. Additionally, apps such as Collector and Explorer use this functionality to integrate with Portal.
 
 ## How to use the sample
+
 When prompted, enter your ArcGIS Online credentials.
 
 ## How it works
-The `DefaultAuthenticationChallengeHandler` is the default implementation of the `AuthenticationChallengeHandler` interface to handle  all security types that ArcGIS supports (including OAuth).  This sample uses it to authenticate with a portal.  This is set on the `AuthenticationManager`. To authenticate the user you create a `Portal` object, provide the essential credentials and then asynchronously load the portal as shown below:
-
-```java
-// Set the DefaultAuthenticationChallengeHandler to allow authentication with the portal.
-DefaultAuthenticationChallengeHandler handler = new DefaultAuthenticationChallengeHandler(this);
-AuthenticationManager.setAuthenticationChallengeHandler(handler);
-// Set loginRequired to true always prompt for credential,
-// When set to false to only login if required by the portal
-final Portal portal = new Portal("http://www.arcgis.com", true);
-```
-
-The `Portal` class follows the loadable pattern and includes listeners to monitor the status of loading the portal.
-
-```java
-portal.addDoneLoadingListener(new Runnable() {
-    @Override
-    public void run() {
-        if (portal.getLoadStatus() == LoadStatus.LOADED) {
-        // loaded
-    }
-});
-```
-
-Once the portal has loaded you can access information about the portal and authenticated user.  
-
-```java
-// Get the authenticated portal user
-PortalUser user = portal.getUser();
-// get the users full name
-String userName = user.getFullName();
-```
+
+1. A `Portal` is created, and supplied a `Credential` which uses OAuth in user mode. 
+2. When the app launches, the portal is loaded, which triggers an authentication challenge.
```

Comment:
For steps 2 and 3, I believe since this sample uses DefaultAuthenticationChallengeHandler, all of this is done automatically. So there is no authenticationview, but it's probably worth keeping step 2 in.